src/app/project/board/dataset-metadata.service.ts

```diff
@@ -2,25 +2,24 @@ import { Injectable } from '@angular/core';
 import { IId, Organization, Person } from '@dasch-swiss/dsp-js';
 
 @Injectable({
-  providedIn: 'root'
+    providedIn: 'root'
 })
-export class DatasetMetadataService {
+export class MetadataService {
 
-  constructor() { }
+    constructor() { }
 
-  /**
-  * used in project metadata page
-  * determine if the object is of type Person or Organization or Iid
-  * @param obj Person | Organization | IId
-  */
-  getContactType(obj: Person | Organization | IId): string {
-    if (obj instanceof Person) {
-      return 'person';
+    /**
+     * used in project metadata page
+     * determine if the object is of type Person or Organization or Iid
+     * @param obj Person | Organization | IId
+     */
+    getContactType(obj: Person | Organization | IId): string {
+        if (obj instanceof Person) {
+            return 'person';
+        } else if (obj instanceof Organization) {
+            return 'organization';
+        }
+        return undefined;
```

Comment:
ending `else` is missing here

Reply:
From my understanding if the case is default, we can add it without using else block. isn't it right?

Reply:
Correct.